controllers/authentication.py

```diff
@@ -33,6 +35,7 @@
 
 class Register(Resource):
 
+    @requires_auth
```

Comment:
Nothing in this file should be annotated with `@requires_auth`

Reply:
Hi,remove the  @requires_auth on the authentication file

Reply:
> Hi,remove the @requires_auth on the authentication file